charts/prometheus-kafka-exporter/templates/deployment.yaml

```
@@ -7,6 +7,9 @@ metadata:
     chart: {{ template "prometheus-kafka-exporter.chart" . }}
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
+    {{- with .Values.labels }}
```

I'd suggest to be more specific.

```suggestion
    {{- with .Values.deploymentLabels }}
```